assets/js/components/UI/PreviewItems.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import PropTypes from "prop-types";
+import UIWorkImage from "./Image";
+import { getImageUrl } from "../../services/helpers";
+
+/** @jsx jsx */
+import { css, jsx } from "@emotion/core";
+
+const inlineList = css`
+  white-space: nowrap;
+  overflow: auto;
+  scrollbar-color: #999 #ccc;
+  scrollbar-width: thin;
+  ::-webkit-scrollbar {
+    height: 0.5rem;
+    background-color: #ccc;
+  }
+  ::-webkit-scrollbar-track {
+    border-radius: 1rem;
+  }
+  ::-webkit-scrollbar-thumb {
+    background: #999;
+    border-radius: 1rem;
+  }
+`;
+
+const previewItem = css`
+  opacity: 1;
+  transition: 0.3s;
+
+  &:hover {
+    opacity: 0.75;
+  }
+`;
+
+export default function PreviewItems({ items = [] }) {
```

Comment:
Following our naming convention pattern, this should be `export default function UIPreviewItems({ items = [] }) {`